raiden/tests/unit/test_wal.py

```diff
@@ -43,7 +43,7 @@ def state_transtion_acc(state, state_change):
 
 
 def new_wal(state_transition: Callable, state: State = None) -> WriteAheadLog:
-    serializer = JSONSerializer
+    serializer = JSONSerializer()
```

Comment:
Wow, how did these test work?

Reply:
The used methods are static methods, so it doesn't make a difference.